src/request_list.js

```diff
@@ -812,7 +821,7 @@ export class RequestList {
  *
  *   If `null`, the list will not be persisted and will only be stored in memory. Process restart
  *   will then cause the list to be crawled again from the beginning. We suggest always using a name.
- * @param {Array<RequestOptions|Request|string>} sources
+ * @param {RequestListOptions['sources']} sources
```

Comment:
lets see how this will work with the old docs (it should work fine with the sdk 3 ts api docs, but here - god knows)